mx.sulong/mx_sulong.py

```diff
@@ -675,4 +680,5 @@ def getResults(self):
     'lli' : [runLLVM, ''],
     'travis1' : [travis1, ''],
     'travis2' : [travis2, ''],
+    'inject' : [injectBitcode, ''],
```

Comment:
Maybe call the command `inject-bc` or `inject-bitcode` so that its purpose is self-explanatory?